gq/gq_iso_test.go

```diff
@@ -94,9 +88,7 @@ func TestVerifierISO(t *testing.T) {
 	encodedSigISO := util.Base64EncodeForJWT(sigISO)
 	ok := svISO.Verify(encodedSigISO, idISO, mISO)
 
-	if !ok {
-		t.Fatal("signature verification failed")
-	}
+	require.True(t, ok, "signature verification failed")
```

Comment:
Since you are touching this code, might as well fix the lowercase message. I looked it up because I wasn't sure if require messages should should start with uppercase or lowercase and they are uppercase.

```suggestion
	require.True(t, ok, "Signature verification failed")
```